lib/degreed/content/courses.rb

```diff
@@ -52,6 +56,43 @@ def all(external_id: nil)
         request.get(content_courses_url, params: params)
       end
 
+      # Update course
+      #
+      # @see https://api.degreed.com/docs/#update-a-specific-course
+      #
+      # @param title [String]
+      # @param external_id [String, #to_s]
+      # @param url [String]
+      # @param duration [Integer, #to_i]
+      # @param duration_type [String]
+      #   Accepted values: Seconds, Minutes, Hours, or Days
+      # @param summary [String] Optional (default: nil)
+      #
+      # @return [Degreed::Response]
+      def update(id:, **kwargs)
+        allowed_attributes = %w[
+          title
+          url
+          duration
+          duration-type
+          obsolete
```

Comment:
Just to make sure I'm connecting the right dots, but does this PR utilize the changes made in [this branch](https://github.com/lessonly/degreed/pull/2/files) in the degreed gem? We don't have `obsolete` included in `allowed_attributes` in the update over there, but I might be missing something.

Reply:
Good question @b-ee 🤔 @corymichaelhack Let me know if I'm off base but I think #2 has been closed in favor of this PR.

Reply:
That's is correct, [#2](https://github.com/lessonly/degreed/pull/2/files) is closed. Adam had started a branch and I started one after him forgetting he had one open. So we continued with this one.

Reply:
OH I was totally on the wrong track of mind. Yup. cool.